.travis.yml

```diff
@@ -1,24 +1,62 @@
 language: dart
-sudo: false
+sudo: required
+
+# We require PhantomJS 2.x, which is only available on Trusty.
+dist: trusty
 
 dart:
  - dev
  - stable
  - 1.23.0
  - 1.22.1
 
+# Content shell needs these fonts.
+addons:
+  apt:
+    packages:
+    - ttf-kochi-mincho
+    - ttf-kochi-gothic
+    - ttf-dejavu
+    - ttf-indic-fonts
+    - fonts-tlwg-garuda
+
+before_install:
+  # Content shell needs this font. Since it has a EULA, we need to manually
+  # install it.
+  #
+  # TODO: remove this and use "sudo: false" when travis-ci/travis-ci#4714 is
+  # fixed.
+  - sudo apt-get update -yq
+  - sudo sh -c "echo ttf-mscorefonts-installer msttcorefonts/accepted-mscorefonts-eula select true | debconf-set-selections"
+  - sudo apt-get install msttcorefonts -qq
+
+  - mkdir -p bin
+  - export PATH="$PATH:`pwd`/bin/"
+  - ln -s `which chromium-browser` bin/google-chrome
+
+  - wget "http://gsdview.appspot.com/dart-archive/channels/stable/release/latest/dartium/content_shell-linux-x64-release.zip"
+  - unzip content_shell-linux-x64-release.zip
+  - ln -s `pwd`/`echo drt-linux-*`/content_shell bin/content_shell
+
 dart_task:
+ # Browser tests take particularly long on Dartium, so we split them up into different tasks.
  - test: --platform dartium
    install_dartium: true
- - test: --platform vm -x content-shell -x chrome -x firefox -x dartium -x phantomjs --timeout 4x
- - dartfmt
- - dartanalyzer
+ - test: --platform vm -x browser
+ - test: --platform vm -t content-shell -x chrome -x firefox -x phantomjs -x dartium --timeout 4x
```

Comment:
...Er, I guess this is testing the vm side of the integration – so it only matters that content_shell is usable...right?